rest_framework_bulk/mixins.py

```diff
@@ -26,15 +25,16 @@ def create(self, request, *args, **kwargs):
             return super(BulkCreateModelMixin, self).create(request, *args, **kwargs)
 
         else:
-            serializer = self.get_serializer(data=request.DATA, many=True)
+            serializer = self.get_serializer(data=request.data, many=True)
             if serializer.is_valid():
-                [self.pre_save(obj) for obj in serializer.object]
-                self.object = serializer.save(force_insert=True)
-                [self.post_save(obj, created=True) for obj in self.object]
+                self.perform_create(serializer)
                 return Response(serializer.data, status=status.HTTP_201_CREATED)
 
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
+    def perform_create(self, serializer):
```

Comment:
`perform_create` is provided by the generics API, and it's most likely the primary override point for users who previously used the old save hooks. I'd recommend creating a new hook, `perform_bulk_create` and embedding logic in there to call `perform_create` if it exists, or fall  back to the old `pre_save` and `post_save` loops if it doesn't.